src/svg-quirks-mode/fonts/LICENSE.txt

```diff
@@ -0,0 +1,202 @@
+
+                                 Apache License
+                           Version 2.0, January 2004
+                        http://www.apache.org/licenses/
```

Comment:
I'm not an expert here, but if I'm reading the [chart on the License Compatibility page on Wikipedia](https://en.wikipedia.org/wiki/License_compatibility#Compatibility_of_FOSS_licenses) correctly it seems we like we can't include Apache 2.0-licensed work in this repository. I'm even less sure about the OFL below.